data/modules/Rondel/Rondel.lua

```diff
@@ -11,14 +11,15 @@ local ShipDef = require 'ShipDef'
 local SystemPath = require 'SystemPath'
 local Timer = require 'Timer'
 local Commodities = require 'Commodities'
+local PlayerState = require 'PlayerState'
 
 local ShipBuilder = require 'modules.MissionUtils.ShipBuilder'
 local OutfitRules = ShipBuilder.OutfitRules
 
 --local Character = require 'Character'
 
 local l_rondel = Lang.GetResource("module-rondel")
-local l_ui_core = Lang.GetResource("ui-core")
+local lui = Lang.GetResource("ui-core")
```

Comment:
Isn't this line unused?

Reply:
Yes. I could have removed it, but chose to leave it there (in canonical format) for future improvements to the module.

In this particular case, the rationale is that it makes it easier for new/junior developers to follow our format/conventions when they're referencing Rondel as an example gameplay module. I'm not too concerned about the performance cost.